packages/utils/src/instrument/fetch.ts

```diff
@@ -48,12 +48,13 @@ function instrumentFetch(onFetchResolved?: (response: Response) => void, skipNat
 
   fill(GLOBAL_OBJ, 'fetch', function (originalFetch: () => void): () => void {
     return function (...args: any[]): void {
-      const { method, url } = parseFetchArgs(args);
+      const { method, url, body } = parseFetchArgs(args);
       const handlerData: HandlerDataFetch = {
         args,
         fetchData: {
           method,
           url,
+          body,
```

Comment:
IMO collecting fetch req payload shouldn't be a problem as long as _**only**_ the graphql requests are sampled. Because xhr instrumentation already collects the [payload](https://github.com/getsentry/sentry-javascript/blob/39c0b1da3e306621bf41ddbffeb823ec49729b97/packages/browser-utils/src/instrument/xhr.ts#L129).